packages/office-ui-fabric-react/src/components/Rating/Rating.scss

```diff
@@ -29,6 +29,10 @@ $Rating-largeIconSize: 20px;
         & ~ .star {
           // Set default color for any star after hovered one
           color: $Rating-defaultColor;
+
+          .halfStar{
```

Comment:
This is because we have redefined the css again for half star element. Reason below